OpenMTRDemo/Forms/TiledFiltersForm.cs

```diff
@@ -194,13 +194,13 @@ private void Pane_DblClickHandler(object sender, EventArgs e)
         {
             try
             {
-                ExpandedImageForm expandedImageForm;
+                ExpandedImageDialog expandedImageForm;
                 PictureBox pictureBox = sender as PictureBox;
                 Bitmap image = (Bitmap)pictureBox.Image;
 
                 Mat mat = OpenCvSharp.Extensions.BitmapConverter.ToMat(image);
 
-                expandedImageForm = new ExpandedImageForm(mat);
+                expandedImageForm = new ExpandedImageDialog(_meterPane1.SourceImage, mat);
```

Comment:
I don't see the point  in sending in the virgin source image here. The base image the user double clicks on is the image they will be manipulating so just use that as the source.  If I am manipulating a sobel image I generally don't care what the source image looks like.

Reply:
@Matt how would you go about removing filters, then?